extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessor.java

```diff
@@ -731,10 +731,18 @@ private void performBackup()
     if ( !authResponse.equals( "true" ) ) {
       System.err.println( Messages.getInstance().getString( "CommandLineProcessor.ERROR_0006_NON_ADMIN_CREDENTIALS" ) );
     }
+
+    // Build the complete URL to use
+    String backupURL = contextURL + API_REPO_FILES_BACKUP;
+
+    // BISERVER-14935: In case "requestParameterAuthenticationEnabled" is set to 'true'
+    backupURL += ( "?userid=" + getUsername() + "&password=" + Encr.encryptPassword(
```

Comment:
Here you are creating a string on line 736 and then creating a new one. Why not create the string with all data instead of reassigning a new value or use a StringBuilder.

Reply:
Same as last one: Deprecate StringUtils.isEmpty(Object) and replace remaining usage (e.g. with ObjectUtils.isEmpty)